### PR TITLE
FS-3820 Adding welsh all questions page

### PR DESCRIPTION
**Pull request by adamdavies1**
https://dluhcdigital.atlassian.net/browse/FS-3820

### Change description
Adding all questions page for welsh forms

- [ ] Unit tests and other appropriate tests added or updated
- [ ] README and other documentation has been updated / added (if needed)
- [ ] Commit messages are meaningful and follow good commit message guidelines (e.g. "FS-XXXX: Add margin to nav items preventing overlapping of logo")

### How to test
Test the all questions page

### Screenshots of UI changes (if applicable)
image

### CODE REVIEW

File: app/templates/all_questions/cy/cof_r3w3_all_questions_cy.html [hunk 1]
```diff
@@ -0,0 +1,1634 @@
...
+    1.1.1. Enwau'r sefydliad
+  </h4>
+  <p class="govuk-body">
+    Eich rhif traciwr unigryw
+  </p>
+  <p class="govuk-body">
+    Roedd hwn wedi'i gynnwys yn y llythyr yr anfonom i gadarnhau eich datganiad o ddiddordeb llwyddiannus.
+  </p>
+  <p class="govuk-body">
+    Er enghraifft, â€˜ANON-###-###-###â€™
```
Comment (rayhan15243) [inline app/templates/all_questions/cy/cof_r3w3_all_questions_cy.html:61]: Why are there special characters? shouldn't it just be "ANON-###-###-###" like we got for English?

File: app/templates/all_questions/cy/cof_r3w3_all_questions_cy.html [hunk 2]
```diff
@@ -0,0 +1,1634 @@
...
+  <ul class="govuk-list govuk-list--bullet">
+    <li class="">
+      gwasanaethau cyhoeddus
+    </li>
+    <li class="">
+      staffio
+    </li>
+  </ul>
+  <p class="govuk-body">
+    Cofiwch, gallwch wneud cais am hyd at Â£50,000 o'ch costau refeniw (dim mwy na 20% o'ch cyllid cyfalaf).
```
Comment (rayhan15243) [inline app/templates/all_questions/cy/cof_r3w3_all_questions_cy.html:1310]: Special character here

File: app/templates/all_questions/cy/cof_r3w3_all_questions_cy.html [hunk 3]
```diff
@@ -0,0 +1,1634 @@
...
+  <ul class="govuk-list govuk-list--bullet">
+    <li class="">
+      gwasanaethau cyhoeddus
+    </li>
+    <li class="">
+      staffio
+    </li>
+  </ul>
+  <p class="govuk-body">
+    Cofiwch, gallwch wneud cais am hyd at ddeg mil o bunnoedd (Â£50,000) o'ch costau refeniw (dim mwy na 20% o'ch cyllid cyfalaf).
```
Comment (rayhan15243) [inline app/templates/all_questions/cy/cof_r3w3_all_questions_cy.html:1281]: Special character here; Â£50,000

File: app/templates/all_questions/cy/cof_r3w3_all_questions_cy.html [hunk 4]
```diff
@@ -0,0 +1,1634 @@
...
+    </li>
+  </ul>
+  <p class="govuk-body">
+    Cofiwch, gallwch wneud cais am hyd at 80% (neu 90% os yw'r darparwr cymorth datblygu wedi cadarnhau eich bod yn gymwys i wneud hyn) o'ch costau cyfalaf, hyd at gwerth o Â£2 filiwn.
+  </p>
+  <p class="govuk-body">
+    Gallwch ddefnyddio eich cynllun busnes i ddarparu gwybodaeth sy'n cefnogi eich atebion.
+  </p>
+  <p class="govuk-body">
+    You can add multiple costau cyfalaf
```
Comment (rayhan15243) [inline app/templates/all_questions/cy/cof_r3w3_all_questions_cy.html:1114]: This is partially in English

File: app/templates/all_questions/cy/cof_r3w3_all_questions_cy.html [hunk 5]
```diff
@@ -0,0 +1,1634 @@
...
+      Sefydliad corfforedig elusennol (CIO)
+    </li>
+    <li class="">
+      Cwmni cydweithredol, fel cymdeithas budd cymunedol
+    </li>
+    <li class="">
+      Cwmni buddiannau cymunedol (CIC)
+    </li>
+    <li class="">
+      Company limited by guarantee
```
Comment (rayhan15243) [inline app/templates/all_questions/cy/cof_r3w3_all_questions_cy.html:185]: This is displaying in English.

File: app/templates/all_questions/cy/cof_r3w3_all_questions_cy.html [hunk 6]
```diff
@@ -0,0 +1,1634 @@
...
+    Os 'Arall', ewch i <strong>1.1.3.1</strong>
+  </p>
+  <p class="govuk-body">
+    Os 'Cwmni cydweithredol, fel cymdeithas budd cymunedol', ewch i <strong>1.1.3.2</strong>
+  </p>
+  <p class="govuk-body">
+    Os 'Sefydliad corfforedig elusennol (CIO)', ewch i <strong>1.1.3.3</strong>
+  </p>
+  <p class="govuk-body">
+    Os 'Parish, town or community council', ewch i <strong>1.1.4</strong>
```
Comment (rayhan15243) [inline app/templates/all_questions/cy/cof_r3w3_all_questions_cy.html:207]: This is displaying in English.

File: app/templates/all_questions/cy/cof_r3w3_all_questions_cy.html [hunk 7]
```diff
@@ -0,0 +1,1634 @@
...
+    4.1. Cyllido angenrheidiol
+  </h3>
+  <h4 class="govuk-heading-s">
+    4.1.1. Cais cyllido cyfalaf
+  </h4>
+  <p class="govuk-body">
+    Costau cyfalaf yw costau prynu neu leastu eich ased a thalu am adnewyddu.
+  </p>
+  <p class="govuk-body">
+    Bydd Gronfa Perchen Cymunedol (COF) yn cyllido hyd at 80% o gostau cyfalaf cyfanswm eich prosiect, hyd at uchafswm o Â£2 filiwn.
```
Comment (rayhan15243) [inline app/templates/all_questions/cy/cof_r3w3_all_questions_cy.html:1058]: Special character displayed here: " Â£2 filiwn."

File: app/templates/all_questions/cy/cof_r3w3_all_questions_cy.html [hunk 8]
```diff
@@ -0,0 +1,1634 @@
...
+  <p class="govuk-body">
+    Rhaid i 20% o gostau cyfalaf cyfanswm eich prosiect gael eu talu gan ffynonellau eraill (cyllid cydweddu). Er enghraifft:
+  </p>
+  <ul class="govuk-list govuk-list--bullet">
+    <li class="">
+      Rydych am brynu ased cymunedol sy'n costio Â£2,500,000. Gallwch wneud cais am Â£2m o COF, a rhaid i chi godi'r gweddill o
+Â£500,000 o ffynonellau eraill.
+    </li>
+    <li class="">
+      Rydych am brynu ased cymunedol sy'n costio Â£250,000. Gallwch wneud cais am Â£200,000 o COF, a rhaid i chi godi'r gweddill o Â£50,000 o ffynonellau eraill.
```
Comment (rayhan15243) [inline app/templates/all_questions/cy/cof_r3w3_all_questions_cy.html:1069]: Special character displayed here: " Â."

File: app/templates/all_questions/cy/cof_r3w3_all_questions_cy.html [hunk 9]
```diff
@@ -0,0 +1,1634 @@
...
+    Faint o gyllid oâ€™r grant COF y byddech yn ei ddefnyddio i dalu am y gost hon?
+  </p>
+  <p class="govuk-body">
+    Faint oâ€™r arian cyfatebol fyddwch chiâ€™n ei ddefnyddio i dalu am y gost hon?
+  </p>
+  <h4 class="govuk-heading-s">
+    4.1.3. Os byddwch wedi sicrhau cyllid cydweddu
+  </h4>
+  <p class="govuk-body">
+    Rhaid i 20% oâ€™ch costau cyfalaf cyfanswm gael eu talu gan eich mudiad neu gyllidwr arall (cyllid cydweddu).
```
Comment (rayhan15243) [inline app/templates/all_questions/cy/cof_r3w3_all_questions_cy.html:1132]: Special characters displayed here "€™"

File: app/templates/all_questions/cy/cof_r3w3_all_questions_cy.html [hunk 10]
```diff
@@ -0,0 +1,1634 @@
...
+    Disgrifiwch y gost
+  </p>
+  <p class="govuk-body">
+    Swm
+  </p>
+  <p class="govuk-body">
+    Faint o gyllid oâ€™r grant COF y byddech yn ei ddefnyddio i dalu am y gost hon?
+  </p>
+  <p class="govuk-body">
+    Faint oâ€™r arian cyfatebol fyddwch chiâ€™n ei ddefnyddio i dalu am y gost hon?
```
Comment (rayhan15243) [inline app/templates/all_questions/cy/cof_r3w3_all_questions_cy.html:1126]: Special characters displayed here "€™"

File: app/templates/all_questions/cy/cof_r3w3_all_questions_cy.html [hunk 11]
```diff
@@ -0,0 +1,1634 @@
...
+    You can add multiple costau cyfalaf
+  </p>
+  <p class="govuk-body">
+    Disgrifiwch y gost
+  </p>
+  <p class="govuk-body">
+    Swm
+  </p>
+  <p class="govuk-body">
+    Faint o gyllid oâ€™r grant COF y byddech yn ei ddefnyddio i dalu am y gost hon?
```
Comment (rayhan15243) [inline app/templates/all_questions/cy/cof_r3w3_all_questions_cy.html:1123]: Special characters displayed here "€™"

File: app/templates/all_questions/cy/cof_r3w3_all_questions_cy.html [hunk 12]
```diff
@@ -0,0 +1,1634 @@
...
+    Ffynhonnell gyllid wedi'i sicrhau
+  </p>
+  <p class="govuk-body">
+    Swm
+  </p>
+  <h4 class="govuk-heading-s">
+    4.1.3.2. A ydych di ddylu bydd chi wedi defnyddio yr cyllid cydweddu y chydwybododd chi?
+  </h4>
+  <p class="govuk-body">
+    Noder na fydd unrhyw gyllid rydych eisoes wediâ€™i wario yn gymwys fel cyllid cydweddu
```
Comment (rayhan15243) [inline app/templates/all_questions/cy/cof_r3w3_all_questions_cy.html:1220]: Special characters displayed here "€™"

File: app/templates/all_questions/cy/cof_r3w3_all_questions_cy.html [hunk 13]
```diff
@@ -0,0 +1,1634 @@
...
+  <ul class="govuk-list govuk-list--bullet">
+    <li class="">
+      dweud wrthym faint o amser y bydd yn cymryd i sicrhau unrhyw gyllid cydweddu sydd ar Ã´l gennych
+    </li>
+    <li class="">
+      darparu unrhyw wybodaeth sy'n cefnogi eich atebion
+    </li>
+  </ul>
+  <p class="govuk-body">
+    You can add multiple cyllid cydweddu heb ei sicrhau
```
Comment (rayhan15243) [inline app/templates/all_questions/cy/cof_r3w3_all_questions_cy.html:1255]: This is partially displaying in English.

File: app/templates/all_questions/cy/cof_r3w3_all_questions_cy.html [hunk 14]
```diff
@@ -0,0 +1,1634 @@
...
+    1.1.6. Ceisiadau ar y cyd
+  </h4>
+  <p class="govuk-body">
+    A yw eich cais yn gynnig ar y cyd mewn partneriaeth Ã¢ sefydliadau eraill
+  </p>
+  <p class="govuk-body">
+    Os 'Ydy', ewch i <strong>1.1.6.1</strong>
+  </p>
+  <h4 class="govuk-heading-s">
+    1.1.6.1. Manylion sefydliad partner
```
Comment (rayhan15243) [inline app/templates/all_questions/cy/cof_r3w3_all_questions_cy.html:326]: Is there a Welsh translation for "partner"?

File: app/templates/all_questions/cy/cof_r3w3_all_questions_cy.html [hunk 15]
```diff
@@ -0,0 +1,1634 @@
...
+  </h4>
+  <p class="govuk-body">
+    Enw'r sefydliad partner
+  </p>
+  <p class="govuk-body">
+    Cyfeiriad y sefydliad partner
```
Comment (rayhan15243) [inline app/templates/all_questions/cy/cof_r3w3_all_questions_cy.html:332]: Is there a Welsh translation for "partner"?

File: app/templates/all_questions/cy/cof_r3w3_all_questions_cy.html [hunk 16]
```diff
@@ -0,0 +1,1634 @@
...
+    1.2. Gwybodaeth am yr ymgeisydd
+  </h3>
+  <h4 class="govuk-heading-s">
+    1.2.1. Manylion y prif gyswllt
+  </h4>
+  <p class="govuk-body">
+    Enw'r cyswllt arweiniol
+  </p>
+  <p class="govuk-body">
+    Bydd yn derbyn yr holl wybodaeth am y cais hwn..
```
Comment (rayhan15243) [inline app/templates/all_questions/cy/cof_r3w3_all_questions_cy.html:353]: Remove extra full stop.

**Review by RamuniN — APPROVED** [review]
LGTM

File: app/templates/all_questions/cy/cof_r3w3_all_questions_cy.html (hunk 5 above)
Comment (adamdavies1) [reply app/templates/all_questions/cy/cof_r3w3_all_questions_cy.html:185]: This is another missing translation which we need to make a ticket for

File: app/templates/all_questions/cy/cof_r3w3_all_questions_cy.html (hunk 6 above)
Comment (adamdavies1) [reply app/templates/all_questions/cy/cof_r3w3_all_questions_cy.html:207]: This is another missing translation which we need to make a ticket for

File: app/templates/all_questions/cy/cof_r3w3_all_questions_cy.html (hunk 14 above)
Comment (adamdavies1) [reply app/templates/all_questions/cy/cof_r3w3_all_questions_cy.html:326]: Missing translation for partner

Comment (rayhan15243) [reply app/templates/all_questions/cy/cof_r3w3_all_questions_cy.html:326]: https://dluhcdigital.atlassian.net/browse/FS-3862 ticket created

File: app/templates/all_questions/cy/cof_r3w3_all_questions_cy.html (hunk 6 above)
Comment (rayhan15243) [reply app/templates/all_questions/cy/cof_r3w3_all_questions_cy.html:207]: https://dluhcdigital.atlassian.net/browse/FS-3862 ticket created.

File: app/templates/all_questions/cy/cof_r3w3_all_questions_cy.html [hunk 17]
```diff
@@ -0,0 +1,1634 @@
...
+      Cwmni buddiannau cymunedol (CIC)
+    </li>
+    <li class="">
+      Company limited by guarantee
+    </li>
+    <li class="">
+      Sefydliad corfforedig elusennol yr Alban (SCIO)
+    </li>
+    <li class="">
+      Parish, town or community council
```
[the remark this answers is not in the document]
Comment (rayhan15243) [reply app/templates/all_questions/cy/cof_r3w3_all_questions_cy.html:191]: https://dluhcdigital.atlassian.net/browse/FS-3862 ticket created.

File: app/templates/all_questions/cy/cof_r3w3_all_questions_cy.html (hunk 5 above)
Comment (rayhan15243) [reply app/templates/all_questions/cy/cof_r3w3_all_questions_cy.html:185]: https://dluhcdigital.atlassian.net/browse/FS-3862 ticket created.

File: app/templates/all_questions/cy/cof_r3w3_all_questions_cy.html [hunk 18]
```diff
@@ -0,0 +1,1634 @@
...
+  <p class="govuk-body">
+    Rhaid i 20% o gostau cyfalaf cyfanswm eich prosiect gael eu talu gan ffynonellau eraill (cyllid cydweddu). Er enghraifft:
+  </p>
+  <ul class="govuk-list govuk-list--bullet">
+    <li class="">
+      Rydych am brynu ased cymunedol sy'n costio £2,500,000. Gallwch wneud cais am £2m o COF, a rhaid i chi godi'r gweddill o
+£500,000 o ffynonellau eraill.
+    </li>
+    <li class="">
+      Rydych am brynu ased cymunedol sy'n costio £250,000. Gallwch wneud cais am £200,000 o COF, a rhaid i chi godi'r gweddill o Â£50,000 o ffynonellau eraill.
```
Comment (rayhan15243) [inline app/templates/all_questions/cy/cof_r3w3_all_questions_cy.html:1069]: One still there " Â£50,000"

**Review by rayhan15243 — APPROVED** [review]
Looks all good now

**Pull request merged**